templates/f5/0/rancher-compose.yml

```diff
@@ -0,0 +1,34 @@
+.catalog:
+  name: “F5 LB”
+  version: "v0.1.1-rancher1"
+  description: "Rancher External LB service powered by F5 BigIP."
+  questions:
+    - variable: "F5_BIGIP_HOST"
+      label: "F5 BIGIP HOST"
```

Comment:
These labels could be friendlier and match the BigIP-casing, they don't need to match the variable names.